frontend/src/employee-frontend/components/messages/SingleThreadView.tsx

```diff
@@ -192,29 +216,54 @@ export function SingleThreadView({
       </ContentArea>
       <Gap size="xs" />
       <ScrollContainer>
+        {title && type && (
+          <StickyTitleRow>
+            <H2>{title}</H2> <MessageTypeChip type={type} />
+          </StickyTitleRow>
+        )}
         {messages.map((message, idx) => (
-          <SingleMessage
-            key={message.id}
-            message={message}
-            title={idx === 0 ? title : undefined}
-            type={idx === 0 ? type : undefined}
-            index={idx}
-            onAttachmentUnavailable={onAttachmentUnavailable}
-          />
+          <>
+            {!replyEditorVisible && idx === messages.length - 1 && (
+              <div style={{ position: 'relative' }}>
+                <span
+                  style={{ position: 'absolute', top: '-100px' }}
```

Comment:
This `-100px` is a little (dirty?) hack to scroll correctly to the top of the last message, if it's very long.